client/subscription/sub-container.js

```diff
@@ -31,6 +32,21 @@ export default class SubContainer extends Component {
       })
     }
   }
+  async unsubscribe({ target }) {
+    const $selected = target.closest('div[ data-id ]')
+    if (!$selected) return
+    const id = $selected.getAttribute('data-id')
+    const confirmed = window.confirm('Do you want to unsubscribe?')
+    if (!confirmed) return
+    const result = await fetch('/subscription/' + id, { method: 'DELETE' })
+    if (confirmed && result) {
```

Comment:
`confirmed` will always be `true` here. You'd have `return`ed otherwise. Also, `result` will always be truthy. This function would have thrown otherwise (aborting the rest of the function).